example_packages/unet2d/unet2d/__main__.py

```diff
@@ -0,0 +1,58 @@
+"""Main module."""
+import torch
+import torch.utils.data
+from tqdm import trange
+
+from bioengine import api
```

Comment:
Currently, __main__.py only works when the engine execute it, and we inject an `api` object to a fake module named `bioengine`. It would be great if we can make this file work alone. We can perhaps make an api module, and bootstrap the engine during the import.